examples/lock-app/nrfconnect/README.md

```diff
@@ -101,135 +100,116 @@ the nRF52840's NFC antenna. Moreover in the reaction on sensing field from the
 smartphone CHIP device will start BLE advertisement, what is an alternative way
 to trigger this from manually pressing Button 4.
 
-<a name="building"></a>
+## Setting up environment
 
-## Building
+First, checkout CHIP repository and sync submodules using the following command:
 
-### Using Docker container
+        $ git submodule update --init
 
-> **Important**:
->
-> Due to
-> [certain limitations of Docker for MacOS](https://docs.docker.com/docker-for-mac/faqs/#can-i-pass-through-a-usb-device-to-a-container)
-> it is impossible to use the Docker container to communicate with a USB device
-> such as nRF 52840 DK. Therefore, MacOS users are advised to follow the
-> [Using Native shell](#using-native-shell) instruction.
+The example requires nRF Connect SDK v1.4.0. You can either install it along
+with related tools directly on your system or use a Docker image which comes
+with the tools pre-installed.
 
-The easiest way to get started with the example is to use nRF Connect SDK Docker
-image for CHIP applications. Run the following commands to start a Docker
-container:
+Note that if you're a macOS user, you won't be able to use the Docker container
+to flash the application onto a Nordic board due to
+[certain limitations of Docker for macOS](https://docs.docker.com/docker-for-mac/faqs/#can-i-pass-through-a-usb-device-to-a-container),
+so it's recommended that you skip to the
+[Using native shell](#using-native-shell) instruction.
+
+### Using Docker container
+
+If you don't have nRF Connect SDK installed yet, create a directory where it
+should be placed:
 
         $ mkdir ~/nrfconnect
-        $ mkdir ~/connectedhomeip
+
+Download the latest `nordicsemi/nrfconnect-chip` Docker image:
+
         $ docker pull nordicsemi/nrfconnect-chip
+
+The next command will start a Docker container using the image acquired in the
+previous step. Please read the below explanation for all specified arguments
+before proceeding.
+
         $ docker run --rm -it -e RUNAS=$(id -u) -v ~/nrfconnect:/var/ncs -v ~/connectedhomeip:/var/chip \
             -v /dev/bus/usb:/dev/bus/usb --device-cgroup-rule "c 189:* rmw" nordicsemi/nrfconnect-chip
 
 > **Note**:
 >
+> -   `~/connectedhomeip` should be replaced with an absolute path to CHIP
```

Comment:
In other places you use <dir_path> style to show that argument needs to be replaced, so it would be good to align it here and keep one style in whole document.